packages/back-end/src/services/features.ts

```diff
@@ -977,7 +978,15 @@ export function getApiFeatureObj({
       featureEnvironments[env].definition = JSON.stringify(definition);
     }
   });
-
+  const revision = await getRevision(
+    feature.organization,
+    feature.id,
+    feature.version
+  );
+  const publishedBy =
+    revision?.publishedBy?.type === "api_key"
+      ? "API"
```

Comment:
@jdorn not sure if we can get this data but would we wait the API key name here?